lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -254,7 +283,7 @@ class LighthouseReportViewer {
     try {
       const json = JSON.parse(e.clipboardData.getData('text'));
       this._reportIsFromGist = false;
-      this._replaceReportHTML(json);
```

Comment:
lol, so this was just broken?

Reply:
> lol, so this was just broken?

ha, yes. No idea for how long...given that it is (or appears to be) a change from `replaceReportHTML` -> `replaceReportHtml`, I would give good odds it was my fault when converting to 2.0 last year...